application/templates/static_site/measure.html

```diff
@@ -1,5 +1,14 @@
 {% extends "base.html" %}
 
+{% set breadcrumbs = 
+  [
+    {"url": url_for("static_site.index"), "text": "Ethnicity facts and figures", "event": "breadcrumb-homepage"},
+    {"url": url_for("static_site.topic", uri=topic), "text": measure_page.parent.parent.title, "event": "breadcrumb-" + topic},
+  ]
+  if not preview else
```

Comment:
This doesn't quite seem to be replicating current breadcrumb behaviour on the "Edit measure" page (at least, I think it's this template) - the CMS has full breadcrumbs on the edit measure page but changes here lose the first breadcrumb - eg:

![screenshot from 2018-10-01 09-38-56](https://user-images.githubusercontent.com/6525554/46278532-4d57d680-c55e-11e8-93a4-7157bd7eee21.png)

![screenshot from 2018-10-01 09-39-08](https://user-images.githubusercontent.com/6525554/46278541-50eb5d80-c55e-11e8-8569-4843f7e7725e.png)

Reply:
Aha, OK I see now (next commit) that this is deliberate.  Ignore me.